client/src/components/CafeInfoModal.tsx

```diff
@@ -0,0 +1,103 @@
+import { useState } from 'react';
+import { styled } from 'styled-components';
+import { SlArrowDown } from 'react-icons/sl';
+
+type CafeInfoModal = {
+  title: string;
+  address: string;
+  content: string;
+};
+
+const CafeInfoModal = ({ title, address, content }: CafeInfoModal) => {
```

Comment:
```suggestion
type CafeInfoModalProps = {
  title: string;
  address: string;
  content: string;
};

const CafeInfoModal = ({ title, address, content }: CafeInfoModalProps) => {
```
`~Props`로 명시하는 것이 좋지 않을까요?

Reply:
알겠습니당